forceng.js

```diff
@@ -305,7 +356,54 @@ angular.module('forceng', [])
      *  data:  JSON object to send in the request body - Optional
      */
     function request(obj) {
-
+        // NB: networkPlugin will be defined only if login was done through plugin and container is using Mobile SDK 5.0 or above
+        if (false && networkPlugin) { 
```

Comment:
Hi @mattgoldspink yes, for SalesforceSDK5, it will use Cordova NetworkPlugin API instead of angular requests, but it is not totally working for me yet. So I have to turn it off for now.